--- /home/arkanon/git/dipohlo/build/0.11.0/share/scripts/load_icons.tcl.orig	2015-06-17 21:33:13.000000000 -0300
+++ /home/arkanon/git/dipohlo/build/0.11.0/share/scripts/load_icons.tcl	2015-08-23 19:33:07.927825910 -0300
@@ -194,17 +194,17 @@
 				set fallback_off "led-off.png"
 			}
 			throttle {
-				set image_on     ""
-				set image_off    "${icon}.png"
-				set fallback_on  ""
-				set fallback_off ""
+				set image_on     "led-off.png"
+				set image_off    "led-on.png"
+				set fallback_on  "led-off.png"
+				set fallback_off "led-on.png"
 				set current_fade_delay_non_active($icon) 0
 			}
 			default {
-				set image_on     "${icon}.png"
-				set image_off    ""
-				set fallback_on  ""
-				set fallback_off ""
+				set image_on     "led-on.png"
+				set image_off    "led-off.png"
+				set fallback_on  "led-on.png"
+				set fallback_off "led-off.png"
 				set fade_delay_active($icon) 0
 			}
 		}
@@ -309,7 +309,7 @@
 if {[lindex [openmsx_info setting scale_factor] 2 1] == 1} {
 	user_setting create string osd_leds_set "Name of the OSD icon set" "handheld"
 } else {
-	user_setting create string osd_leds_set "Name of the OSD icon set" "set1"
+	user_setting create string osd_leds_set "Name of the OSD icon set" ""
 }
 user_setting create string osd_leds_pos "Position of the OSD icons" "default"
 set load_icons::current_osd_leds_set $osd_leds_set
